src/index.html

```diff
@@ -11,7 +11,202 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <fieldset>
+      <legend>Personal information</legend>
+
+      <form
+        action="http://localhost:8080/api"
+        method="POST"
+        onsubmit ="onSubmit()"
+      >
+        <div class="form-field_1">
```

Comment:
```suggestion
        <div class="form-field">
```
Зачем тут 1? можешь использовать :first-child, если тебе надо стилизовать первый элемент отдельно